ThemeDecorator/ThemeDecorator.js

```diff
@@ -239,7 +248,22 @@ const ThemeDecorator = hoc(defaultConfig, (config, Wrapped) => {
 		static displayName = 'ThemeDecorator';
 
 		static propTypes = {
+			/**
+			 * A custom accent color, as a hex string.
+			 *
+			 * @memberof agate/ThemeDecorator.ThemeDecorator.prototype
+			 * @type {String}
+			 * @public
+			 */
 			accent: PropTypes.string,
+
```

Comment:
the accent prop is shown twice on document. I don't know reason.

Reply:
I know why So, I can fix it but I don't know how to be documented yet.

Reply:
@ybsung Please help us

Reply:
Let's remove @memberof and convert @public to private in `CustomizableSkinStyle`'s propTypes JS Docs.